packages/cli/src/main/kotlin/elide/tool/cli/cmd/repl/ToolShellCommand.kt

```diff
@@ -487,8 +441,8 @@ import org.graalvm.polyglot.Engine as VMEngine
   // Active line reader.
   private val lineReader: AtomicReference<LineReader?> = AtomicReference(null)
 
-  // All VFS configurators.
-  @Inject lateinit var vfsConfigurators: List<GuestVFS.VFSConfigurator>
+  // Intrinsics manager
+  @Inject internal lateinit var intrinsicsManager: IntrinsicsManager
```

Comment:
Intrinsics are not injected automatically by the core API so we use the manager directly when installing the plugins.